site/en/userGuide/tools/attu.md

```diff
@@ -18,7 +18,10 @@ Attu is under rapid development and new features are added on a weekly basis. We
 
 Below is the features we have to offer:
 
-- View milvus cluster statistics in a glance. (TBD)
+- Support username, password, and TLS connection
+  ![Attu_Login](../../../../assets/attu/insight_install.png "Connect to the Attu service.")
+  ![Attu_Login_user_pwd](../../../../assets/attu/insight_install_user_pwd.png "Connect to the Attu service with user")
```

Comment:
![Attu_Login_user_pwd](../../../../assets/attu/insight_install_user_pwd.png "Connect to the Attu service with username and password.")